ToDoTime/Startup.cs

```diff
@@ -20,11 +32,19 @@ public void ConfigureServices(IServiceCollection services)
 #pragma warning disable CS8603 // Possível retorno de referência nula.
             services.AddScoped<IToDoTimeSQLContext>(provider => provider.GetService<ToDoTimeSQLContext>());
 #pragma warning restore CS8603 // Possível retorno de referência nula.
+
+            services.AddControllers().AddJsonOptions(options =>
```

Comment:
At this point this is not needed, the only reason was the entity was being returned from the backend when fetching, using DTOs fixed it.

Reply:
Removed